src/components/UsersTableComponent.js

```diff
@@ -42,18 +43,18 @@ const columns = [
     formatter: (rowContent, row) => {
       return (
         <div style={{ display: "flex", justifyContent: "space-around" }}>
-          <Link to={"edit/"+row.id}>
-          <Button color="primary">
-            <FontAwesomeIcon icon={faEdit} />
-          </Button>
+          <Link to={"edit/" + row.id}>
```

Comment:
better use string template to decrease complexity @azukhrufy

Reply:
ok got it, thank you